site/.go-version

```diff
@@ -1 +1 @@
-1.19
+1.21.5
```

Comment:
Note: it was necessary to specify the patch version (.5) for Netlify to actually use the version. Specifying `1.21` resulted in the version not being found and falling back to the default 1.19 version used to build the website.